google-account-plugin/src/com/google/cloud/tools/intellij/AccountPluginInitializationComponent.java

```diff
@@ -53,9 +56,26 @@ public void initComponent() {
     }
     if (!ApplicationManager.getApplication().isUnitTestMode()) {
       configureUsageTracking();
+      initUsageTracker();
     }
   }
 
+  @VisibleForTesting
+  void initUsageTracker() {
+    String platformPrefix = System.getProperty("idea.platform.prefix");
+    String trackingId = new PropertiesFilePluginFlags().getAnalyticsId();
+
+    Predicate<Object> userOptedIn = new Predicate<Object>() {
+      @Override
+      public boolean apply(@javax.annotation.Nullable Object object) {
+        return UsageTrackerManager.getInstance().hasUserOptedIn();
+      }
+    };
+
+    IdeaUsageTracker usageTracker = ServiceManager.getService(IdeaUsageTracker.class);
+    usageTracker.init(platformPrefix, trackingId, userOptedIn);
```

Comment:
Perhaps, instead of `Predicate`, passing `PropertiesComponent` can work. (However, I actually want to use a Java 8 lambda function for this, since it will look simple and do the job while looking cool(?).)